src/prepack-cli.js

```diff
@@ -170,6 +174,7 @@ function run(
       }
       process.exit(1);
     }
+    if (serialized == null) serialized = {};
```

Comment:
When `!serialized`, then none of the outputs produced below would make sense. Not sure if it can even happen at this point that `!!serialized`.